_posts/kmorbitzer/2017-05-22-kathryn-turtle-hack.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+author: kmorbitzer
+title: "Kathryn's Turtle Exercise"
+---
+
+Here's the program I'm embedding.
+
+<iframe src="https://trinket.io/embed/python/50aec82a6c" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Here's my reflection.
+
+For my Turtle program, I decided to create a castle.  This allowed me to continue to use many of the skills that we previously learned, as well as find new ways to apply them.  
+
+To start creating the program, I first drew out what I wanted it to ultimately look like.  I then moved the starting point for the castle down from 0,0 to give enough room to build the tower with the flags.  I started by building the left tower first with the triangles, then moved over to the right tower.  After building the castle with the two towers, I next moved onto adding the door and windows.  In order to create the door, I had to first make a circle and color it in black, then create a rectangle over the circle.  By also coloring that in black, it makes both shapes appear to be one.  I then created the windows in a similar way, only on a smaller scale.  I created the flags last, starting with the ones on the left tower and then finishing with the ones on the right tower.  
+
+One of the biggest challenges for me in creating this Turtle program was making sure everything was symmetrical by keeping everything the same height.  I would often have to look back at my previous code to remember how high up I made the tower, the window, and the flags to make sure that I made the ones on the other side the same height.  This made me recognize that I need to come up with a process to keep track of this more efficiently and that I should label the code around certain objects as I’m writing it.    
```

Comment:
Nice observation.  In fact, that's the main piece of feedback I have for you: leave yourself some comments!  It must've been hard reading through your code without them.